data/projects.yml

```diff
@@ -251,7 +251,8 @@ project_list:
     customization_level: extensive
     github_user:
       - sheltonsjma
-    note: "This publication is in honor of the museum's 50th anniversary. The collection catalogue has been extensively customized, including changes to the theme template and significant alterations to layout and functionality of the object entry pages."
+      - egardner
+    note: "Quire is optimized for longevity and discoverability by allowing output in multiple formats from plain-text files and print-on-demand PDF files. These factors were critical in the Museum's decision to use this platform for a milestone anniversary publication and offered a timely opportunity to examine sustainability in the digital publishing realm. In determining what type of software to use for the Museum's fiftieth-anniversary catalogue, concerns around ongoing maintenance, future migration, and accessibility were carefully weighed against available options, especially considering the catalogue's purpose as a testament to the history and evolution of the collection. The San José Museum of Art customized this project to be more visually driven, including new templates for displaying artwork on a single page in a grid and removing the banner/hero image at the top of the page."
```

Comment:
I would make a few small changes:

- from plain-text files to print-on-demand PDF files
- These factors were critical in the San Jose Museum of Art's decision to use this platform 
- The museum customized this project to be less linear and more visually driven, including adding new templates for displaying artwork on a single page in a grid and removing the banner/hero image at the top of the page.